setup.py

```diff
@@ -5,5 +5,5 @@
     packages=['nsqworker', 'locker'],
     version='0.0.23',
     install_requires=['tornado==4.5.3', 'pynsq', 'futures; python_version == "2.7"', 'mdict', 'redis',
-                      'auguryapi @ git+https://github.com/augurysys/auguryapi-py.git@0.9.51'],
+                      'auguryapi @ git+https://github.com/augurysys/auguryapi-py.git@scopes-api-addition'],
```

Comment:
You need to create a master tag for this and then merging.

Reply:
Updated :)